setup.py

```diff
@@ -39,6 +39,10 @@ def prerelease_local_scheme(version):
         'Development Status :: 3 - Alpha',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python :: 3.6'
+        'Programming Language :: Python :: 3.7'
+        'Programming Language :: Python :: 3.8'
+        'Programming Language :: Python :: 3.9'
+        'Programming Language :: Python :: 3.10'
```

Comment:
Avogadro currently does have wheels for 3.10, so we should probably hold off on adding that one. Otherwise looks good, thank you.